pkg/datasources/parameters_acceptance_test.go

```diff
@@ -104,15 +107,25 @@ func TestAcc_Parameters_QuotedIdentifiersIgnoreCaseCanBeSet(t *testing.T) {
 		},
 		Steps: []resource.TestStep{
 			{
-				Config: `resource "snowflake_account_parameter" "test" {
-				  key   = "QUOTED_IDENTIFIERS_IGNORE_CASE"
-				  value = "true"
-				}`,
+				Config: sessionParameterOnUser(userId.Name()),
 			},
 		},
 	})
 }
 
+func sessionParameterOnUser(userName string) string {
+	return fmt.Sprintf(
+		`
+	resource "snowflake_user" "u" {
```

Comment:
There is no "rule" if we should prefer helper over tf config. The rule of thumb should be: what is more readable? In this case, the user resource is really simple to set up, so both options would be good. It's a bit hard now to get the id of the helper-generated user to put it into config (because of how tf tests are run - we will address this in the future, maybe on one of the happy Fridays), and as you can see there is no method to create with name (and these methods should be avoided for various cases, mostly because it opens a way to use id that is not the generated one), that's why I went with user in tf config.